en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md

```diff
@@ -37,12 +37,12 @@ In the `[controlPlane.eventHub]` section,
   serviceUrl = "https://192.168.1.94:9443/"
   username="admin"
   password="$env{cp_admin_pwd}"
-  environmentLabels = ["Production and Sandbox"]
+  environmentLabels = ["Default"]
   retryInterval = 5
   skipSSLVerification=true
   # Message broker connection URL of the control plane
   [controlPlane.eventHub.jmsConnectionParameters]
-    eventListeningEndpoints = ["amqp://admin:$env{cp_admin_pwd}@apim:5672?retries='10'&connectdelay='30'"]
+    eventListeningEndpoints = ["amqp://admin:$env{cp_admin_pwd}@192.168.1.94:5672?retries='10'&connectdelay='30'"]
```

Comment:
Why is there a specific IP address here

Reply:
Updated this with a placeholder and removed the specific ip